src/main/java/com/gizasystems/automationexercise/pages/HomePage.java

```diff
@@ -34,6 +34,12 @@ public HomePage clickOnScrollUpArrow() {
         return this;
     }
 
+    @Step("Scroll to top page")
+    public HomePage scrollToTopPage() {
+        driver.element().scrollToElement(slideHeading_div);
+        return this;
+    }
+
```

Comment:
We need to discuss the scrolling tests as some cases require clicking on the arrow itself and the arrow button is not part of the Home Page.

Reply:
@Step("Click on scroll up arrow")
    public HomePage clickOnScrollUpArrow() {
        driver.element().click(scrollUpArrow_button);
        return this;
    }
    
    private final By scrollUpArrow_button = By.xpath("//i[@class='fa fa-angle-up']");
    
   I think you mean this method and the locator of the arrow are not part of the Home page.
   You are right, let's discuss this next meeting